frontend/src/lib/uPlotLib/getUplotChartOptions.ts

```diff
@@ -209,12 +212,12 @@ export const getUPlotChartOptions = ({
 				},
 			],
 		},
-		series: getSeries(
+		series: getSeries({
 			apiResponse,
-			apiResponse?.data.result,
+			widgetMetaData: apiResponse?.data.result,
 			graphsVisibilityStates,
-			fillSpans,
```

Comment:
Why are we removing fillSpans here ?

Reply:
We were only passing it as an argument and not using it inside the function.